uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/document-list_test.js

```diff
@@ -40,6 +46,41 @@ define([
       });
     });
 
+    describe('$scope.changeStatus()', function () {
+      beforeEach(function () {
+        initController();
+        $scope.document = mockDocument;
+      });
+
+      afterEach(function () {
+        $rootScope.$apply();
+      });
+
+      describe('status is empty', function () {
+        beforeEach(function () {
+          $scope.changeStatus($scope.document, null);
+        });
+
+        it('does not call Document service to save the status of the document', function () {
```

Comment:
try to keep the description "conversational" unless you absolutely need to

"it does not update the document"